### DISCUSSION TITLE
How to test the email config?

### DISCUSSION THREAD

**Discussion by greyidol** [Retrospring/retrospring#1309] [Q&A] [answered]
I had previously set up smtp instead of sendmail by modifying config/environments/production.rb, but I lost those changes. Now I think I put them back correctly, but emails aren't sending, and I'm not sure why. Is there any way to test the mailer in the `bundle exec rails c` console?

**Comment by pixeldesu** [comment, marked as answer]
You should be able to create an instance of `Retromail` (our customized Devise mailer, the only mailer we actually use at the moment) in the Rails console and send an email with it.

Not exactly sure if it will work, but worth a shot maybe:

```rb
@mailer = Retromail.new
# you can run `@mailer.smtp_settings` to look/set the currently applied SMTP settings

@mailer.mail(from: 'sender@example.com', to: 'recipient@example.com', subject: 'test', body: "Hello, you've got mail!").deliver
```
